spec/models/state_changer/standard_plate_spec.rb

```diff
@@ -21,12 +21,25 @@
     context 'when the plate is the initial plate in the pipeline' do
       include_context 'a limber target plate with submissions', 'pending'
 
-      let(:target_state) { 'started' }
+      context 'when the target state is started' do
+        let(:target_state) { 'started' }
 
-      it 'starts the requests', :aggregate_failures do
-        # Requests are started and we create one event per order.
-        expect { state_changer.update_labware_state }.to change(BroadcastEvent::LibraryStart, :count).by(1)
-        expect(library_requests).to all(be_started)
+        it 'starts the requests', :aggregate_failures do
+          # Requests are started and we create one event per order.
+          expect { state_changer.update_labware_state }.to change(BroadcastEvent::LibraryStart, :count).by(1)
+          expect(library_requests).to all(be_started)
+        end
+      end
+
+      # RVI BCL uses the processed_1 state as the first state for the the intial plate
```

Comment:
typo intial -> initial

Reply:
My first question would be why doesn't RVI used the 'started' state as the first state? So it sticks to the same convention as others. I forget the details of this pipeline...

My second question is, what happens if a pipeline uses both 'started' and 'processed_1' states? It doesn't trigger LibraryStart twice does it?

Reply:
Good question. The RVI RT plate (the second plate in the pipeline) has 4 operations on it which require bed verifications so it needs to move through a lot of states. The best way to do this was to move it through the various processed states (I think we added processed_3 and processed_4 for this).

In hindsight moving to 'started' first and then the other states might have made more sense, but started is not a required first state. The on_started method is also called when moving from 'pending' to 'passed' which most pipelines do every plate transition.

No pipeline has both started and processed_1 but if they did nothing would happen, once the requests are started they cannot be started again.